site/content/en/latest/api/extension_types.md

```diff
@@ -577,6 +577,7 @@ _Appears in:_
 | Value | Description |
 | ----- | ----------- |
 | `SourceIP` | SourceIPConsistentHashType hashes based on the source IP address.<br /> | 
+| `Header` | HeaderConsistentHashType hashes based on a request header.<br /> | 
```

Comment:
FYI this line still exists in the generated documentation. Seems like it might be generated from the union discriminator validation comments? The `+notImplementedHide` did not hide it, even when added on `HeaderConsistentHashType`.

Reply:
cc @zirain can we disable this as well ?
@harrisonturton thanks for highlighting this, lets not block your PR on this

Reply:
this's expected, you cannot find  `header` under `ConsistentHash`.